{
	"id": "CVE-2019-15796",
	"description": "",
	"hardware": [],
	"software": [
		{
			"name": "python-apt",
			"platform": "debian-9",
			"version": "< 1.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-10",
			"version": "< 1.8.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-1.1",
			"version": "< 1.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-1.2",
			"version": "< 1.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-1.3",
			"version": "< 1.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-2",
			"version": "< 1.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-2.1",
			"version": "< 1.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-2.2",
			"version": "< 1.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-3",
			"version": "< 1.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-3.1",
			"version": "< 1.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-4",
			"version": "< 1.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-5",
			"version": "< 1.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-6",
			"version": "< 1.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-7",
			"version": "< 1.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-8",
			"version": "< 1.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-1.1",
			"version": "< 1.8.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-1.2",
			"version": "< 1.8.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-1.3",
			"version": "< 1.8.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-2",
			"version": "< 1.8.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-2.1",
			"version": "< 1.8.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-2.2",
			"version": "< 1.8.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-3",
			"version": "< 1.8.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-3.1",
			"version": "< 1.8.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-4",
			"version": "< 1.8.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-5",
			"version": "< 1.8.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-6",
			"version": "< 1.8.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-7",
			"version": "< 1.8.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-8",
			"version": "< 1.8.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-9",
			"version": "< 1.8.4.1"
		},
		{
			"name": "python-apt",
			"platform": "debian-12",
			"version": "< 1.8.5"
		},
		{
			"name": "python-apt",
			"platform": "debian-11",
			"version": "< 1.8.5"
		}
	],
	"references": [
		"https://security-tracker.debian.org/tracker/DSA-4609-1"
	],
	"severity": null,
	"state": "invalid",
	"is_edited": false
}